### PR TITLE
Split model classes into separate submodule

### PR DESCRIPTION
**Pull request by mbarnes**
Got myself into a circular dependency situation with some work I'm doing, where `clusters.py` needed models from `host.py` and `host.py` needed models from `cluster.py`.

Python no likie.

The `Model` classes are small and standalone, so I created a `models.py` and lumped them all together there so the `Resource` classes can import them without conflict.

I know there's less drastic workarounds but this seems cleaner for the long term.

What do you think?

### CODE REVIEW

File: src/commissaire/handlers/clusters.py [hunk 1]
```diff
@@ -21,63 +21,9 @@
...
-        'started_at', 'finished_at')
-
-
-class Clusters(Model):
-    """
-    Representation of a group of one or more Clusters.
-    """
-    _json_type = list
-    _attributes = ('clusters',)
+from commissaire.handlers.models import *
```
Comment (ashcrow) [inline src/commissaire/handlers/clusters.py]: nit: I'd like to only import the models used. (same for the other import *'s).

**Comment by ashcrow** [comment]
All in all it looks good. One nit on importing pattern.

File: src/commissaire/handlers/clusters.py (hunk 1 above)
Comment (mbarnes) [inline src/commissaire/handlers/clusters.py]: Thought you might say that.  Will fix.

**Comment by mbarnes** [comment]
Importing fixed.  Please squash and merge if you're okay with that.

**Comment by ashcrow** [comment]
LGTM

**Comment by ashcrow** [comment]
Merged.

**Pull request closed without merging**